projects/igniteui-angular/src/lib/stepper/stepper.component.ts

```diff
@@ -332,10 +334,10 @@ export class IgxStepperComponent extends IgxCarouselComponentBase implements Igx
 
     constructor(
         cdr: ChangeDetectorRef,
```

Comment:
```suggestion
       private cdr: ChangeDetectorRef,
```

Reply:
this is already marked as private in base class